ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -71,9 +71,39 @@ conformance_test(
     test_tool_args = [
         "--verbose",
         "--concurrent-test-runs=4",  # lowered from default #procs to reduce flakes - details in https://github.com/digital-asset/daml/issues/7316
-        "--exclude=ContractKeysIT" +  # 3 of the more specialized tests currently fail on Canton
+        # The following three contract key tests require uniqueness and are tested in conformance-test-contract-keys below
+        "--exclude=ContractKeysIT,ContractKeysIT:CKFetchOrLookup,ContractKeysIT:CKNoFetchUndisclosed,ContractKeysIT:CKMaintainerScoped" +
         ",ConfigManagementServiceIT,LedgerConfigurationServiceIT" +  # dynamic config management not supported by Canton
         ",TransactionScaleIT,LotsOfPartiesIT" +  # scale tests need timeout-scale-factor 1.5 and 2.0 respectively on Canton
```

Comment:
I think we deleted these two tests. Would you mind seeing whether deleting them causes any problems?